var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -147,13 +173,13 @@ class Cp2k(MakefilePackage, CudaPackage):
             )
 
     with when("+libxc"):
-        depends_on("pkgconfig", type="build", when="@7.0:")
-        depends_on("libxc@2.2.2:3", when="@:5", type="build")
-        depends_on("libxc@4.0.3:4", when="@6.0:6.9", type="build")
+        depends_on("pkgconfig", when="@7.0:")
```

Comment:
not sure about the others (I see that a discussion on this topic has already occurred with @dev-zero), but probably `pkgconfig` is actually needed just at build time...isn't it?